docs/innkrevingsoppdrag/beskrivelse-av-tjenester.md

```diff
@@ -212,4 +213,25 @@ Tjenesten returnerer for en gitt kravidentifikator:
     - Det opprinnelige kravet
     - Eventuelle endringer på kravet, med transaksjonsid
     - Eventuell avskriving av kravet, med transaksjonsid
- 
\ No newline at end of file
+
+## Se kravdetaljer
+
+Tjenesten gir oppdragsgiver mulighet til å hente ut detaljer om et eksisterende krav ved å gjøre et GET-kall til 
+endepunktet `/api/innkreving/innkrevingsoppdrag/v1/innkrevingsoppdrag/{kravidentifikator}`. Det finnes også et 
+mock-endepunkt i test tilgjengelig på 
+`/api/innkreving/innkrevingsoppdrag/v1/innkrevingsoppdrag/{kravidentifikator}/mock` (Se "Kravdetaljer" i
+[SwaggerHub](https://app.swaggerhub.com/apis-docs/skatteetaten/oppdragsinnkreving-api/1.15.115#/Kravdetaljer)).
+
+I kallet til endepunktet må oppdragsgiver angi parameter "Kravidentifikatortype" som må være ett av alternativene:
```

Comment:
Tenker det holde med en litt enklere og mindre teknisk beskrivelse, siden utviklere har tilgang til Swagger. Har pushet en ny beskrivelse av dette "Oppdragsgiver må angi hvilken type kravidentifikator som skal brukes. For krav mottatt gjennom det nye kravmottaket, kan Skatteetatens kravidentifikator benyttes. For alle andre krav kan oppdragsgivers egen kravidentifikator brukes."